WebHostLib/tracker.py

```diff
@@ -328,20 +372,71 @@ def getPlayerTracker(tracker: UUID, tracked_team: int, tracked_player: int, want
                     if recipient == tracked_player:  # a check done for the tracked player
                         attribute_item_solo(inventory, item)
                     if ms_player == tracked_player:  # a check done by the tracked player
-                        checks_done[location_to_area[location]] += 1
-                        checks_done["Total"] += 1
-    specific_tracker = game_specific_trackers.get(games[tracked_player], None)
-    if specific_tracker and not want_generic:
+                        lttp_checks_done[location_to_area[location]] += 1
+                        lttp_checks_done["Total"] += 1
+
+    if tracked_player in prog_items:
+        pass
+    else:
+        prog_items[tracked_player] = collections.Counter()
+        for player in locations:
+            for location in locations[player]:
+                item, recipient, flags = locations[player][location]
+                if recipient == tracked_player:
+                    if flags & 1:
+                        item_name = lookup_any_item_id_to_name[item]
+                        prog_items[tracked_player][item_name] += 1
+
+    game_name = games[tracked_player]
+    # TODO move all games in game_specific_trackers to new system
```

Comment:
TODO. When is this getting done and by whom?

Reply:
I'd definitely prefer for the original developers of those trackers do it as they can control the format to their original intent though I suppose we'll see how that goes.

Reply:
Any movement on this?